### ISSUE TITLE
Error message when report metadata has invalid json

### ISSUE THREAD

**Issue by Hikariii** [Block8/PHPCI#540]
The ```meta_value``` field in the ```build_meta``` table is a text field and not big enough to contain large plugin reports. The data got truncated and I got invalid json.
I solved it by changing the field type for the ```meta_value``` field:

```sql
ALTER TABLE  `build_meta` CHANGE  `meta_value`  `meta_value` LONGTEXT CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL ;
```

I suggest at least error handling for invalid json, so that a user sees that his errors are too much for phpci.

It would also be nice to enlarge the field for big reports.

**Comment by Hikariii** [comment]
The corresponding phinx issue https://github.com/robmorgan/phinx/issues/249 is merged https://github.com/robmorgan/phinx/issues/249 some time ago and included in the newest versions.

**Comment by dancryer** [comment]
Actually the fix has been merged but not included in any releases yet. We're waiting on the next release, I believe 0.5.0 for the fix to be included.

**Comment by Hikariii** [comment]
It seems the implementation differs from the PR, but from the release notes and the mysql adapter code it looks like longtext just works:
- https://github.com/robmorgan/phinx#version-history (v0.4.0)
- https://github.com/robmorgan/phinx/blob/v0.4.2.1/src/Phinx/Db/Adapter/MysqlAdapter.php#L720-L735

**Comment by dancryer** [comment]
Our fix for this issue has been merged into master and will be included in the next release (1.6.0)

**Comment by amenk** [comment]
Is still don't see PHP Unit output. What can be the reason?

**Comment by mavimo** [comment]
@amenk can you fill a new issue with all information to help us to debug the issue you have? Thx :)
